instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
> I imagine: Turn left at the end of the road instead of In 200 meters, turn left at the end of the road.

@MoKob correct, however this repo will not have any idea about distances. When this module is used by the developer, it's their responsibility to add the distances.